layouts/team/single.html

```diff
@@ -89,8 +89,12 @@ <h3 class="mb-5">{{ i18n "courses" | upper}}</h3>
           <h3 class="section-title">{{ i18n "blogs_by" }} {{ .Title }}</h3>
         </div>
       </div>
-      {{ $posts := where site.RegularPages "Type" "post" }}
-      {{ $posts := $posts | intersect (where $posts "Params.author" .Title )}}
+      {{$posts := slice}}
+      {{range where site.RegularPages "Type" "post"}}
```

Comment:
Don't know for sure if replacing this will be better in the long run

As a comparison:
```suggestion
{{ $page_link := .Permalink }}
{{ $tags := .Params.tags }}
{{ range .Site.Pages }}
    {{ $page := . }}
    {{ $has_common_tags := intersect $tags .Params.tags | len | lt 0 }}
    {{ if and $has_common_tags (ne $page_link $page.Permalink) }}
        <li><a href="{{ $page.Permalink }}">{{ $page.Title }}</a></li>
    {{ end }}
{{ end }}
```